### ISSUE TITLE
trim query + append query (without =) hangs

### ISSUE THREAD

**Issue by bagder** [curl/trurl#80]
With current git master (bf3efc5770f30d0806c) this *hangs* the tool!
~~~
$ trurl "https://example.com?search=hello" --trim query=search --append query=moo

**Comment by paulroub** [comment]
It's actually less-specific than that. The same failure is seen if you run

```sh
trurl "https://example.com?search=hello&moo" --trim query=search
```

**Comment by bagder** [comment]
Fixed via 3e8ce1d9913b5a55
